### PR TITLE
Devel: adds abbott generation

### LINKED ISSUE #18: Generate a typical time history for abbott generation.
This issue can be closed when a typical time history of APP's generation has been produced.

### PR DESCRIPTION
**Pull request by samgdotson**
Closes #18 by generating a typical time history for abbott generation.

### CODE REVIEW

**Comment by samgdotson** [comment]
Closes #20 by adding synthetic histories

File: data_processing/abbott_generation.ipynb [hunk 1]
```diff
@@ -0,0 +1,1092 @@
...
...PPi8OBlFMh3OQElUg0zqyYnh4lU5hkhW576sX544VJJ14tJ/Tvh1L6VCjcoAAzu2j7h9P1EWBAQEowPoVjOTa1yqn53Ay79y3wAUoTKHYeO4+I/f+FbWb3Elv3HFLL0JVucRbxMd72xdk9zRV0VRvYsywZTyCKpsCVe5+Xl4cc//jFee+01vP7665gxYwauvPJKv8uWRYLQRrp69JLhtp9tjYm41GKj7CUOHJNcR+iG3Q3rRnM0hqhIlMUhP8f+7lFgFmACYngyqVMCyLc5PaUdUx0P1XvNydWm90puS97vAgwFRA8X1zeW7TDVb7FKNW15LhktuWW52YjsskkQa5S0meKBADhnaBdJwaQjEZf+8k4R9x1t5pp2sydOPIoas3DwNJsI4wJlJEjyXFgWbzlomLYVWI4n/Y9Qhm9LVMTHP5ngOh+3sKOw1fL79JV5xliXsqAhFW52PATRoseozb/ackD13SrS2fq95taJduA/ubi/6ScL7FQ5c/G2hBRPc2TqgbteW4mzh3Sx7bZnRbA77al5+OnMFa7LReFlFMgbXlziWVpW2LyvkRvAQoujza14ccEW3Da5Vs8Bp6EMcCLTeM2TRAhBt9ICDOpaklA6guZLOlm07TjcpAq+YgatTNEaI0p/eGnht1i94wi+2uxelkg26D7DLILvwcYW7GnIrEADuu0LAfZkgylkkWS4ns3nz5/vZTmy8AEPvbPW9bMnPPihhyWxxn0y+TJdCOp3H3UsWDdrTH8LcyO4a0p/288TjWRkJ39GbWI7HwB49LK4EpDlFeBhWI9SlBd7z6ditOQ6dcOyklvMhDE2L+2Go1omg3TD8bSEUahYKSB5m1Xe5ueLDfuk+wlB/87t8I/pY7jv/s8FW3QcT2bwYgPJpiEqFk/qhHnWIM0Oo7qwkBS08YiQgFQ379w2XnW6/eWmA4EiWj7IKG5Ya7jdR5rQub1k0WM3ElZbRJ9KeVwG0KLHaNh9tUmteJrUz5xEePaKxKxJOpXk4YHZaxJKo61Au/FNhOOpOyegiJ2p+CyGUF2Zy5jfl28zjszaFnDP6yttHVIcPh7FZ/XSOsmjJRAYy3LDJcGngzYtmqIx7Dqc2KabfQcBkgvftoPHEytYEtESFW2NDyr78BBiLL/TRfH2hswfKhKikgdY3PXaCjwlW0IW50Vw6egeSSufX+BZdto9pM0iC
```
Comment (robfairh) [inline data_processing/abbott_generation.ipynb]: I think this is not the actual demand, but the output power of abbot.

Comment (robfairh) [inline data_processing/abbott_generation.ipynb:1039]: Same comment as before.

**Review by robfairh — CHANGES_REQUESTED** [review]
See minor comments. The rest looks good.

**Comment by robfairh** [comment]
Looks good

**Pull request merged**